src/ui/manager.tsx

```diff
@@ -30,7 +30,9 @@ const WebReaderContent: React.FC<ReaderReturn> = ({
       {navigator && state && (
         <Header readerState={state} navigator={navigator} />
       )}
-      <PageButton onClick={navigator?.goBackward} left={0}>{`<`}</PageButton>
+      {state && !state.isScrolling && (
```

Comment:
this is the reason why I don't think we should re-use the page buttons - the "sensible UI" for epubs is that, in scrolling mode, the page buttons don't appear.

Reply:
Yea I guess in both PDFs and EPUBs no matter which mode you are in, you need a forward and back button though, and to me it seemed sensible to have just one forward/back button. But yea we don't have design guidance so it doesn't really matter. If you want to separate them then we can separate them, but they should be rendered here in the manager, not in the reader content component itself